lighthouse-core/test/audits/speed-index-metric-test.js

```diff
@@ -74,7 +74,6 @@ describe('Performance: speed-index-metric audit', () => {
     const artifacts = mockArtifactsWithSpeedlineResult(SpeedlineResult);
 
     return Audit.audit(artifacts).then(response => {
-      assert.equal(response.displayValue, '845');
```

Comment:
> looks like you nuked it because `.displayValue` is never explictly set?

yes. That line in `audit.js` does the same thing as it did before (so the final result output by Lighthouse will have a `displayValue` of `'845'`), but since the unit test is calling `.audit()` directly, it's not getting the version run through `generateAuditResult` and so won't have `displayValue` on it